main/zoe/guide/price-authority.md

```diff
@@ -48,13 +48,11 @@ requested.
 
 ## Mutable price quotes
 
-A `PriceQuote` is a record (has fields rather than methods), while `MutableQuote` is
-an object (methods, no fields). `MutableQuote`'s method `getPromise(`) returns 
-a `Promise` for a `PriceQuote`, which is the same `Promise` returned by the `quoteWhenLTE()`
-API method and its siblings. 
-
-You can cancel a `MutableQuote` or change its trigger levels to specified values without
-requiring a second `Promise`. 
+`MutableQuote`'s method `getPromise(`) returns a `Promise` for a `PriceQuote`, 
+which is the same `Promise` returned by the `quoteWhenLTE()` API method and its siblings. 
+Effectively, the non-mutable price quote methods return a static `PriceQuote`, while
+the mutable price quote methods return a reusable price quote object which can manipulated 
+by changing its trigger levels or cancelling it. 
```

Comment:
`MutableQuote`'s method `getPromise()` returns a Promise for a `PriceQuote`,  which is the same as what is returned by the `quoteWhenLTE()` API method and variants.  Effectively, the non-mutable price quote methods return a single `PriceQuote`, while the mutable price quote methods return a reusable object which can be manipulated by changing its trigger levels or by cancelling it.


The backquote after getPromise shouldn't be inside the parens.